exchanges/gateio/gateio_test.go

```diff
@@ -3437,7 +3437,7 @@ func TestGetTimeInForce(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(t, "gtc", ret)
 
-	ret, err = getTimeInForce(&order.Submit{Type: order.Market, FillOrKill: true})
+	ret, err = getTimeInForce(&order.Submit{Type: order.Market, TimeInForce: order.FOK})
```

Comment:
This is having issue I think
https://github.com/thrasher-corp/gocryptotrader/actions/runs/12864603088/job/35863457166?pr=1382#step:10:430